ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-presubmits.yaml

```diff
@@ -511,13 +463,7 @@ presubmits:
         - --artifact-dir=$(ARTIFACTS)
         - --give-pr-author-access-to-namespace=true
         - --secret-dir=/usr/local/e2e-secrets/azure
-        - --target=[output:stable:azure-controllers]
-        - --target=[output:stable:canary]
-        - --target=[output:stable:etcdbackup]
-        - --target=[output:stable:metricsbridge]
-        - --target=[output:stable:startup]
-        - --target=[output:stable:sync]
-        - --target=[output:stable:tlsproxy]
+        - --target=[output:stable:azure]
         - --target=cluster-launch-e2e-azure-reimagevm
         - --template=/usr/local/cluster-launch-e2e-azure-reimagevm.yaml
```

Comment:
Why do you need to literally target the others? CI Operator is supposed to be running the entire dependency tree for you and there is a mechanism to opt into waiting on images for templates automatically.